GitCommands/RevisionReader.cs

```diff
@@ -99,8 +99,8 @@ public sealed class RevisionReader : IDisposable
                 /* Author email    */ "%aE%n" +
                 /* Committer name  */ "%cN%n" +
                 /* Committer email */ "%cE%n" +
-                /* Commit subject  */ "%s%n%n" +
-                /* Commit body     */ "%b";
+                /* Commit subject  */ "%s%n%n"; // +
+                /////* Commit body     */ "%b";
```

Comment:
I didn't meassure the performance difference. This can be reverted. If this gained any performance, it is neglectable.

Reply:
I want to have the Body back, maybe as an option.
Right now the body is missing also in the CommitInfo panel

Reply:
I reverted the change. 
Is there any way to disable the popup with the commit body? Since there is no timer on the tooltip, it is really annoying when all commits have long commit messages. For example, in the linux repository most commits have over 15 lines causing a lot of popups. A timer would be better in my opinion.  Bit off-topic though....

Reply:
More OT: The tool tip is great in my opinion, but it must be limited to a few lines of the commit body too in addtion to the timer.